pkg/oc/cli/admin/policy/subject_review.go

```diff
@@ -46,13 +44,16 @@ var (
 const SubjectReviewRecommendedName = "scc-subject-review"
 
 type SCCSubjectReviewOptions struct {
-	sccSubjectReviewClient     securitytypedclient.PodSecurityPolicySubjectReviewsGetter
-	sccSelfSubjectReviewClient securitytypedclient.PodSecurityPolicySelfSubjectReviewsGetter
+	PrintFlags *genericclioptions.PrintFlags
+
+	Printer *policyPrinter
+
+	sccSubjectReviewClient     securityv1client.PodSecurityPolicySubjectReviewsGetter
+	sccSelfSubjectReviewClient securityv1client.PodSecurityPolicySelfSubjectReviewsGetter
```

Comment:
Merge these two into one `securityv1typedclient.SecurityV1Interface`, similarly to what I did for image client in #20847.